06_puppy/mohitnag/main.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	m := MemStore{}
+	puppy := Puppy{ID: 1, Breed: "dog", Colour: "white", Value: "$2"}
+	_ = m.CreatePuppy(puppy)
+	puppy, _ = m.ReadPuppy(1)
+	fmt.Fprint(out, puppy.Colour)
+}
```

Comment:
I don't think it's essential according to the lab requirements but I think it would be good to execute both the sync and the map store functionality from the main... also might be worth initiating the updates and deletes too.

Reply:
Done. initiated all the CRUD operations  in main.